utils/docker/Dockerfile-rpmbuild.sles.12.3

```diff
@@ -0,0 +1,116 @@
+                                     libpmemblk-devel librbd-devel  \
+                                     libnuma-devel
+
+# isa-l
+RUN zypper --non-interactive install yasm
+
+# spdk
+RUN zypper --non-interactive ar --gpgcheck-allow-unsigned -f https://build.hpdd.intel.com/job/daos-stack/job/dpdk/job/master/lastSuccessfulBuild/artifact/artifacts/sles12.3/ dpdk; \
+    zypper --non-interactive ar --gpgcheck-allow-unsigned -f https://build.hpdd.intel.com/job/daos-stack/job/fio/job/master/lastSuccessfulBuild/artifact/artifacts/sles12.3/ fio;   \
+    zypper --non-interactive --gpg-auto-import-keys ref dpdk fio
+RUN zypper --non-interactive install dpdk-devel fio-src libiscsi-devel cunit-devel
+
+# meson
+RUN zypper --non-interactive install python-rpm-macros python3-base
+
+# fuse
+RUN zypper --non-interactive ar --gpgcheck-allow-unsigned -f https://build.hpdd.intel.com/job/daos-stack/job/meson/job/master/lastSuccessfulBuild/artifact/artifacts/sles12.3/ meson; \
+    zypper --non-interactive --gpg-auto-import-keys ref meson
+RUN zypper --non-interactive install meson ninja
+
+# pmdk
+RUN zypper --non-interactive install man libunwind-devel gdb bc
+
+# daos
+RUN zypper --non-interactive ar --gpgcheck-allow-unsigned -f https://build.hpdd.intel.com/job/daos-stack/job/spdk/job/master/lastSuccessfulBuild/artifact/artifacts/sles12.3/ spdk;             \
+    zypper --non-interactive ar --gpgcheck-allow-unsigned -f https://build.hpdd.intel.com/job/daos-stack/job/isa-l/job/master/lastSuccessfulBuild/artifact/artifacts/sles12.3/ isa-l;           \
+    zypper --non-interactive ar --gpgcheck-allow-unsigned -f https://build.hpdd.intel.com/job/daos-stack/job/protobuf-c/job/master/lastSuccessfulBuild/artifact/artifacts/sles12.3/ protobuf-c; \
+    zypper --non-interactive ar --gpgcheck-allow-unsigned -f https://build.hpdd.intel.com/job/daos-stack/job/fuse/job/master/lastSuccessfulBuild/artifact/artifacts/sles12.3/ fuse;             \
+    zypper --non-interactive ar --gpgcheck-allow-unsigned -f https://build.hpdd.intel.com/job/daos-stack/job/pmdk/job/master/lastSuccessfulBuild/artifact/artifacts/sles12.3/ pmdk;             \
+    zypper --non-interactive ar --gpgcheck-allow-unsigned -f https://build.hpdd.intel.com/job/daos-stack/job/raft/job/master/lastSuccessfulBuild/artifact/artifacts/sles12.3/ raft;             \
+    zypper --non-interactive ar --gpgcheck-allow-unsigned -f https://build.hpdd.intel.com/job/daos-stack/job/argobots/job/master/lastSuccessfulBuild/artifact/artifacts/sles12.3/ argbots;      \
+    zypper --non-interactive ar --gpgcheck-allow-unsigned -f https://build.hpdd.intel.com/job/daos-stack/job/cart/job/master/lastSuccessfulBuild/artifact/artifacts/sles12.3/ cart;             \
+    zypper --non-interactive ar http://download.opensuse.org/repositories/devel:/languages:/go/SLE_12_SP3_Backports/devel:languages:go.repo;                                                    \
```

Comment:
Need to have the URL passed for the current Jenkins.

Reply:
Fixed.